test/jdk/sun/net/InetAddress/nameservice/simple/CacheTest.java

```diff
@@ -59,8 +60,7 @@ public static void main(String args[]) throws Exception {
             return;
 
         }
-        String hostsFileName = System.getProperty("test.src", ".") + "/CacheTestHosts";
-        System.setProperty("jdk.net.hosts.file", hostsFileName);
+        String hostsFileName = System.getProperty("jdk.net.hosts.file");
```

Comment:
With the "jdk.net.hosts.file" property specified at runtime, a Hosts File for this test is now instead generated in JTwork/scratch. This does not appear to change the behaviour of the test. However, I would appreciate if someone could verify that this is ok for me. Otherwise I can track back to the original solution.